<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Preprocessing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="original_image_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>370</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="processed_image_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>370</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>Processed Image</string>
    </property>
   </widget>
   <widget class="QComboBox" name="preprocessing_options">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_load_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
